gamemodes/horde/gamemode/sv_commands.lua

```diff
@@ -99,6 +99,18 @@ function End(ply)
     ply:PrintMessage(HUD_PRINTTALK, "Stopping game...")
 end
 
+function NearBeacon(ply)
```

Comment:
this should not be a global function
```suggestion
local function nearBeacon(ply)
```